### PR TITLE
Changes from Signup QA

### PR DESCRIPTION
**Pull request by rakvat**
https://3.basecamp.com/3585816/buckets/4848453/todolists/669697144

### CODE REVIEW

File: locales/de.yml [hunk 1]
```diff
@@ -108,7 +108,7 @@ de:
     invitations_sent: Versandte Einladungen
     labels:
       friends_email: E-Mail-Adresse des Freundes
-      friends_email_sub: '%{count} invitations remaining'
+      friends_email_sub: "%{count} invitations remaining"
```
Comment (ErebusBat) [inline locales/de.yml]: Why quotes here, but on [`bg L122`](https://github.com/fetlife/translations/pull/130/files#diff-d193b3555bb7daf1d7234cf5d39e7463R122) and [`de L122`](https://github.com/fetlife/translations/pull/130/files#diff-065b4c643bed6132e5d7bd356083203dR122) you switched from single quotes => no quotes?

File: locales/de.yml [hunk 2]
```diff
@@ -313,13 +313,17 @@ de:
     error_city_missing: Die Stadt muss gewählt werden.
     error_country_missing: Das Land muss gewählt werden.
     error_email: Wir haben Probleme, die obige E-Mail-Adresse zu verarbeiten. Wir empfehlen Dir, die Schreibweise zu überprüfen oder eine andere Adresse zu versuchen.
-    error_gender_required: Gender muss gewählt werden
+    error_email_taken: Email has already been taken.
+    error_gender_required: Gender muss gewählt werden.
     error_nickname: Der Name ist schon vergeben.
+    error_nickname_bad_word: Nickname can't have the word '%{word}' in it.
+    error_nickname_characters: Nickname can only contain letters, numbers, dashes and underscores; no spaces.
+    error_nickname_length: Nickname has to be between 2 and 16 characters long.
```
Comment (ErebusBat) [inline locales/de.yml]: @rakvat Did you not want to translate these five new phrases?

File: locales/de.yml (hunk 1 above)
Comment (rakvat) [reply locales/de.yml]: @ErebusBat I did no do that. Running the i18n tasks must have changed that.

File: locales/de.yml (hunk 2 above)
Comment (rakvat) [reply locales/de.yml]: @ErebusBat recently a member PMed me and asked if they could help with the German translation. I wanted to leave some strings to translate here to potentially empower members. :)

**Pull request merged**